recipes/plink2/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: plink2
+  version: "1.90b3.35"
+
+source:
+  git_url: https://github.com/chrchang/plink-ng/
+  git_rev: 8990fc72295b299c1ace05266861884d319cf082
```

Comment:
Can you please use the github archive to download a tar-ball. We are planning to strore all tarballs on depot.galaxyproject.org and this would make it much easier to implement.